src/Bootstrap/Configurator.php

```diff
@@ -112,6 +115,17 @@ public function addParameters(array $params)
 
 
 	/**
+	 * Add instances of services.
+	 * @return self
+	 */
+	public function addServices(array $services)
+	{
+		$this->services = DI\Config\Helpers::merge($services, $this->services);
+		return $this;
```

Comment:
In this case is enough classic merge `$this->services = $services + $this->services;`